lib/node-repo.js

```diff
@@ -74,29 +74,32 @@ async function updatePrWithLabels (options, labels) {
   }
 }
 
-function removeLabelFromPR (options, label, cb) {
+async function removeLabelFromPR (options, label) {
   // no need to request github if we didn't resolve a label
   if (!label) {
-    return cb()
+    return
   }
 
   options.logger.debug('Trying to remove label: ' + label)
 
-  githubClient.issues.removeLabel({
-    owner: options.owner,
-    repo: options.repo,
-    number: options.prId,
-    name: label
-  }, (err) => {
-    if (err) {
-      if (err.code === 404) return options.logger.info('Label to remove did not exist, bailing ' + label)
-      options.logger.error(err, 'Error while removing a label')
-      return cb(err, null)
+  try {
+    await githubClient.issues.removeLabel({
+      owner: options.owner,
+      repo: options.repo,
+      number: options.prId,
+      name: label
+    })
+  } catch (err) {
+    if (err.code === 404) {
+      options.logger.info('Label to remove did not exist, bailing ' + label)
+      return
     }
+    options.logger.error(err, 'Error while removing a label')
+    return
```

Comment:
```suggestion
    throw err
```

Reply:
Would probably break the backport stuff because this code path is not well tested

Reply:
It was calling `cb(err, null)` in that case. Isn't throwing the equivalent in an async function?

Reply:
It wasn't, the function didn't take a callback, I added the callback on the first commit to test it a little bit better.

This is how it is called today on master: https://github.com/nodejs/github-bot/blob/c90f7c18742be6566cba666048b5c16c4c1842d0/scripts/attempt-backport.js#L143

On a second thought, I guess throwing here wouldn't break anything, would just leave the rejection unhandled (which we can handle in the script). It still worries me a bit because this function isn't called as part of the backport tests, but I guess it should be fine. Will change it tomorrow to throw